docs/common-tasks/incidents.md

```diff
@@ -0,0 +1,468 @@
+# View and Maintain Incident Records
+
+In this example we'll show you how to view incidents from your console, download them as a json file, followed by acknowledging and deleting them from your console.
+
+Before we get stuck in, it's worth mentioning that we have a host of [Useful Scripts](https://github.com/thinkst/canary-utils) that are worth looking into.
+
+<div class="section-container">
+  <div class="details-content">
+
+We'll start by using the [Unacknowledged-Incidents](/incidents/queries.html#unacknowledged-incidents) API to obtain list of Incidents
```

Comment:
For all the descriptions of links, we can drop the `-` so in this case `Unacknowledged Incidents`